cmd/tools/generate/cisco_counter.tmpl

```diff
@@ -0,0 +1,64 @@
+This document describes how Harvest metrics relate to their relevant Cisco Switch, including:
```

Comment:
This document describes which Cisco switch metrics are collected and what those metrics are named in Harvest, including: